lib/kinetic.jl

```diff
@@ -0,0 +1,78 @@
+
+########################################
+## Chemical Kinetic equations
+########################################
+
+
+@doc """
+Chemical kinetic reaction system
+
+```julia
+ReactionSystem(X, S, R, K)
+```
+
+### Arguments
+
+* `X` : State vector (array of unknowns N x 1)
+* `S` : stoichiometric coefficients for reactants (array M x N)
+* `R` : stoichiometric coefficients for products (array M x N)
+* `K` : reaction rates (array M x 1)
+
+### Example
+
+```julia
+function concentration()
+
+    A0 = 0.25
+    rateA = 0.333
+    rateB = 0.16
+
+    X= { Unknown(A0), Unknown(0.0) }
+
+    S = [ [1, 0] [0, 1] ] ## stoichiometric coefficients for reactants
+    R = [ [0, 1] [1, 0] ] ## stoichiometric coefficients for products
+    K = [rateA , rateB] ## reaction rates
+    
+    return ReactionSystem (X, S, R, K)
+
+end
+y = sim(model())
+
+""" -> type DocKinetic <: DocTag end
```

Comment:
This can be just a regular doc string. Leave off the `type DocKinetic...`. That syntax is just for generic documents and headers.